.gitignore

```diff
@@ -18,9 +18,6 @@ out/
 # tests logging output
 tempto_logs
 
-# tempto local configuration
```

Comment:
now tempto-configuration-local.yaml are committed to repo, so ignoring them in git makes it inconvenient